modules/heat_conduction/test/tests/NAFEMS/transient/T3/nafems_t3_edge2_coarse.i

```diff
@@ -0,0 +1,102 @@
+
+[Mesh]
+  type = GeneratedMesh
+  dim = 1
+  nx = 5
+  xmin = 0.0
+  xmax = 0.1
+  elem_type = EDGE2
+[]
+
+[Variables]
+  [./temp]
+    order = FIRST
+    family = LAGRANGE
+    initial_condition = 0.0
+  [../]
+[]
+
+[Functions]
+  [./temp_func]
+    type = ParsedFunction
+    value = '100.0 * sin(pi*t/40)'
+  [../]
+[]
+
+[BCs]
+  [./FixedTempLeft]
+    type = DirichletBC
+    variable = temp
+    boundary = left
+    value = 0.0
+  [../]
+  [./FunctionTempRight]
+    type = FunctionDirichletBC
+    variable = temp
+    boundary = right
+    function = temp_func
+  [../]
+[]
+
+[Kernels]
+  [./heat]
+    type = HeatConduction
+    variable = temp
+  [../]
+  [./HeatTdot]
+    type = HeatConductionTimeDerivative
+    variable = temp
+  [../]
+[]
+
+[Materials]
+  [./heat]
+    type = HeatConductionMaterial
+    block = 0
+    specific_heat = 440.5
+    thermal_conductivity = 35.0
+  [../]
+  [./density]
+    type = GenericConstantMaterial
+    prop_names = 'density '
+    prop_values = '7200.0 '
+  [../]
+[]
+
+[Preconditioning]
+  [./pc]
+    type = SMP
+    full = True
+  [../]
+[]
+
+[Executioner]
+  type = Transient
+  solve_type = 'PJFNK'
+  petsc_options_iname = '-pc_type -pc_factor_mat_solver_package'
+  petsc_options_value = 'lu     superlu_dist'
+
+  l_max_its = 20
+  l_tol = 1e-5
+
+  nl_max_its = 50
+  nl_rel_tol = 1e-10
+  nl_abs_tol = 1e-12
+
+  start_time = 0.0
+  dt = 1
+  end_time = 32.0
+[]
+
+[Postprocessors]
+  [./target_temp]
+    type = NodalVariableValue
+    variable = temp
+    nodeid = 4
+  [../]
+[]
+
+[Outputs]
+  exodus = true
```

Comment:
One or the other is necessary. I assume exodus = true is for outputting to paraview, but probably not necessary for tests.